src/StatsdClient/Statsd.cs

```diff
@@ -50,7 +50,22 @@ public abstract class Metric : ICommandType
             }
         }
 
-        public class Event : ICommandType
+        public abstract class Command : ICommandType
+        {
+            protected static string EscapeContent(string content)
+            {
+                return content
+                    .Replace("\r", "")
+                    .Replace("\n", "\\n");
+            }
+
+            protected static string TruncateOverage(string str, int overage)
+            {
+                return str.Substring(0, str.Length - overage);
+            }
+        }
+

```

Comment:
I think I might be missing something here, sorry.

What are you meaning by this comment?

Are you saying it is odd that Metric doesn't inherit the new Command class?
I did feel this was a little yuck, those methods could be placed anywhere else to make it cleaner.

I'll hold off on any changes until I can get a little clarification.

Cheers.